tests/putobject_test.py

```diff
@@ -151,6 +151,64 @@ def test_aws_put_in_subdirectory(self):
             os.remove(get_filename)
             assert_command(f'irm -rf {self.bucket_irods_path}/{put_directory}')
 
+    def test_mc_put_in_bucket_root_small_file(self):
+
+        put_filename = inspect.currentframe().f_code.co_name 
+        get_filename = f'{put_filename}.get'
+
+        try:
+            make_arbitrary_file(put_filename, 100*1024)
+            assert_command(f'mc cp {put_filename} s3-api-alice/{self.bucket_name}/{put_filename}',
+                    'STDOUT_SINGLELINE',
+                    'small_file')
+            assert_command(f'iget {self.bucket_irods_path}/{put_filename} {get_filename}')
+            assert_command(f'diff {put_filename} {get_filename}')
+
+        finally:
+            os.remove(put_filename)
+            os.remove(get_filename)
+            assert_command(f'irm -f {self.bucket_irods_path}/{put_filename}')
+
+    def test_mc_put_in_bucket_root_large_file(self):
```

Comment:
It's not clear what this test name is describing. Consider using more words or rephrasing.

Reply:
See above.